### DISCUSSION TITLE
Nest js app not detected by git when installed in turborepo.

### DISCUSSION THREAD

**Discussion by a11yster** [vercel/turborepo#6299] [Help] [answered]
I created a new turborepo using:

npx create-turbo@latest

then in app, i created a nestjs app using :

npm i -g @nestjs/cli
nest new api

but the changes do not reflect in the turborepo when I check git status
it shows the nestjs app as empty, only the name of the folder is shown as a change in git.

<img width="1440" alt="Screenshot 2023-10-27 at 8 39 01 PM" src="https://github.com/vercel/turbo/assets/131473060/7d7564a3-eb96-49ac-b478-6eeef6285f2b">

As you can see,the api folder is empty.

**Comment by mehulkar** [comment, marked as answer]
This is happening because `nest new api` is initializing a git repo inside your existing git repo. You can `rm -rf api/.git` to show the newly generated files inside your current git index. 

To be clear, this is not related to Turborepo, and you would see the same behavior outside of a repo that happens to use turbo.

**Reply by nimeshmaharjan1** [reply]
damn stuck on this since two days why is it not shwoing the .git folder that is so annoying

**Reply by sovetski** [reply]
that is very weird, why it commit for us without asking anything?
